srv/salt/_modules/cephinspector.py

```diff
@@ -6,17 +6,41 @@
 from subprocess import Popen, PIPE
 import json
 
+def _get_disk_id(partition):
+    """
+    Return the disk id of a partition/device, or an empty string if not available.
+    """
+    disk_id_cmd = Popen("find -L /dev/disk/by-id -samefile " + partition + " \( -name ata* -o -name nvme* \)", stdout=PIPE, stderr=PIPE, shell=True)
```

Comment:
Do we care about disks whose names start with "scsi" (full disclosure: I don't really know how this by-id generation stuff works)